covasim/webapp/example_web_data.csv

```diff
@@ -0,0 +1,32 @@
+date,new_diagnoses,new_tests,new_deaths
+2020-03-01,1,24,0
```

Comment:
Do we need a calibration date in the UI? Seem like right now, everything is based on n days from "start". Though, that could be a separate task if this is assuming the first entry is day 0

Reply:
@sayers24 yes, date rather than day index would be great!